jaeger.version

```diff
@@ -2,4 +2,4 @@
 # by default with the Jaeger Operator. This would usually be the latest 
 # stable Jaeger version. When you update this file, make sure to update the
 # deploy/operator.yaml and the docs as well.
```

Comment:
Minor: isn't this comment wrong now? The operator.yaml doesn't need to be updated?